### PR TITLE
Project setup & consumer and producer logic

### PR DESCRIPTION
**Pull request by gdellorusso**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	<organization>
+		<name>PagoPa</name>
+		<url>https://www.pagopa.it</url>
+	</organization>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>com.amazonaws</groupId>
+				<artifactId>aws-java-sdk-bom</artifactId>
+				<version>1.12.414</version>
```
Comment (nttdata-rtorsoli) [inline pom.xml]: Use properties

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-dependencies</artifactId>
+				<version>${spring.cloud.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+			<dependency>
+				<groupId>io.awspring.cloud</groupId>
+				<artifactId>spring-cloud-aws-dependencies</artifactId>
+				<version>2.3.0</version>
```
Comment (nttdata-rtorsoli) [inline pom.xml]: Use properties

File: src/main/java/it/pagopa/interop/probing/caller/consumer/PollingReceiver.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+  @SqsListener(value = "${amazon.sqs.end-point.poll-queue}",
+      deletionPolicy = SqsMessageDeletionPolicy.ON_SUCCESS)
+  public void receiveStringMessage(final String message) throws IOException {
+
+    EserviceContent service = mapper.readValue(message, EserviceContent.class);
+
+    logger.logMessageReceiver(service.getId());
+
+    if (EserviceTechnology.REST.getValue().equals(service.getTechnology().getValue())) {
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/caller/consumer/PollingReceiver.java]: ???

File: src/main/java/it/pagopa/interop/probing/caller/dto/EserviceContent.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+import it.pagopa.interop.probing.caller.util.EserviceTechnology;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class EserviceContent {
+
+  @JsonProperty("id")
+  private String id;
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/caller/dto/EserviceContent.java]: String or UUID?

File: src/main/java/it/pagopa/interop/probing/caller/dto/TelemetryDto.java [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+import it.pagopa.interop.probing.caller.util.EserviceStatus;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class TelemetryDto {
+
+  @JsonProperty("serviceName")
+  private String serviceName;
```
Comment (nttdata-rtorsoli) [inline src/main/java/it/pagopa/interop/probing/caller/dto/TelemetryDto.java:13]: ServiceName or Service?

File: src/main/java/it/pagopa/interop/probing/caller/dto/EserviceContent.java (hunk 4 above)
Comment (gdellorusso) [reply src/main/java/it/pagopa/interop/probing/caller/dto/EserviceContent.java]: is right string, is the identifier of the eservices table

File: src/main/java/it/pagopa/interop/probing/caller/consumer/PollingReceiver.java (hunk 3 above)
Comment (gdellorusso) [reply src/main/java/it/pagopa/interop/probing/caller/consumer/PollingReceiver.java]: useful for future developments, but i will delete it for now

**Review by nttdata-rtorsoli — DISMISSED** [review]

File: src/main/java/it/pagopa/interop/probing/caller/dto/EserviceContent.java (hunk 4 above)
Comment (gdellorusso) [reply src/main/java/it/pagopa/interop/probing/caller/dto/EserviceContent.java]: changed in eserviceRecordId to disambiguate it

**Review by galales — APPROVED** [review]

**Review by nttdata-rtorsoli — APPROVED** [review]

**Pull request merged**